<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_SignUp_Page</class>
 <widget class="QWidget" name="User_SignUp_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 24pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Create new Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(152, 152, 152);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
padding-left: 5px;
border-radius: 5px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(152, 152, 152);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
padding-left: 5px;
border-radius: 5px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(152, 152, 152);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
padding-left: 5px;
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirm_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(152, 152, 152);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
padding-left: 5px;
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_confirm_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>CONFIRM PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Already registered?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="s">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>311</width>
     <height>33</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>108</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 8pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Log in here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/signup_bg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_successful">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(66, 255, 104);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label</zorder>
  <zorder>label_username</zorder>
  <zorder>label_email</zorder>
  <zorder>label_password</zorder>
  <zorder>label_confirm_password</zorder>
  <zorder>label_6</zorder>
  <zorder>s</zorder>
  <zorder>commandLinkButton</zorder>
  <zorder>lineEdit_username</zorder>
  <zorder>lineEdit_email</zorder>
  <zorder>lineEdit_password</zorder>
  <zorder>lineEdit_confirm_password</zorder>
  <zorder>label_successful</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
